HyperionScreenCap/Capture/Dx11ScreenCapture.cs

```diff
@@ -109,13 +123,48 @@ public void Initialize()
 
             CaptureWidth = _width / _scalingFactor;
             CaptureHeight = _height / _scalingFactor;
+            
+            // Initialize duplicator so we can see what the output format is
+            InitDesktopDuplicator();
+
+
+            _minCaptureTime = 1000 / _maxFps;
+            _captureTimer = new Stopwatch();
+            _disposed = false;
+        }
+
+        private void InitDesktopDuplicator()
+        {
+            // We're potentially reinitializing the duplicator, which could change output format
+            // So make sure we reinitialize our textures
+            _stagingTexture?.Dispose();
+            _smallerTexture?.Dispose();
+            _smallerTextureView?.Dispose();
+
+            // Duplicate the output
+            Format[] DesktopFormats = {  Format.R16G16B16A16_Float, Format.B8G8R8A8_UNorm };
```

Comment:
I think we should add R10G10B10A2_UNorm format like: `Format[] DesktopFormats = { Format.R16G16B16A16_Float, Format.B8G8R8A8_UNorm, Format.R10G10B10A2_UNorm };`